src/app/providers.jsx

```diff
@@ -112,7 +120,10 @@ export function Providers({ children }) {
         <Global />
         <QueryClientProvider client={client}>
           <WagmiConfigProvider>
-            <XRPLWalletProvider registerWallets={xrplRegisterWallets}>
+            <XRPLWalletProvider
+              registerWallets={xrplRegisterWallets}
+              autoConnect={false}
```

Comment:
I had to add this because without it Axelarscan asked me to login the moment I visited its home page.